src/modules/ims_ipsec_pcscf/cmd.c

```diff
@@ -670,16 +638,11 @@ int ipsec_create(struct sip_msg* m, udomain_t* d, int _cflags)
     }
 
     // Get security parameters
-    if(pcontact->security_temp == NULL) {
+    if(pcontact->security_temp == NULL || pcontact->security_temp->type != SECURITY_IPSEC) {
```

Comment:
Keep the old code. The new one cannot distinguish between the errors: "No security parameters found" and "Unsupported security type"